pycbc/events/events.py

```diff
@@ -731,6 +742,18 @@ def __setitem__(self, name, data):
                 f['search/setup_time_fraction'] = \
                     numpy.array([float(self.setup_time) / float(self.run_time)])
 
+            if 'q_trans' in self.global_params:
```

Comment:
Let's not add this to them multievent case. You aren't adding into anything that will call this.